packages/webapp/constants/baseAddresses.ts

```diff
@@ -0,0 +1,58 @@
+import { AddressType, CHAIN_IDS } from './types'
+
+export type ContractInfo = {
+  Proxy: string,
+  Implementation: string,
+  Block: number
+}
+
+export type Contracts = {
+  Token: ContractInfo,
+  AuctionHouse: ContractInfo,
+  Governor: ContractInfo,
+  Treasury: ContractInfo,
+  Metadata: ContractInfo
+}
+
+export const contracts: Contracts = {
+  Token: {
+    Proxy: "0x880Fb3Cf5c6Cc2d7DFC13a993E839a9411200C17",
+    Implementation: "0xe77e4fa003b2cc07ad10a9d1db216cae5ed14d3f",
+    Block: 9893263
+  },
+AuctionHouse: {
```

Comment:
fix spacing for `contracts` object

Reply:
Just ran prettier on this file to fix indents and it adjusted the file to the current project to the current .prettierrc rules, which actually changed all the other indents